### DISCUSSION TITLE
glTF Working Group Meeting 2022-02-24

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#68] [General]
# OMI glTF Working Group Meeting 2/24/2022

This meeting will be on 2/24/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- https://github.com/omigroup/gltf-extensions/issues/62
- https://github.com/omigroup/gltf-extensions/pull/67
- https://github.com/omigroup/gltf-extensions/issues/66

- OMI_audio_emitter check-in
  - Vote to advance to stage 2?
- Collider extension check-in
  - https://github.com/omigroup/gltf-extensions/pull/63
- Avatar Animation Retargeting

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by robertlong** [comment]
# Notes
- Dynamic Text
  - Maybe focus more on SDF text?
  - Possibility work with Mozilla / @feiss on a final spec? Or should we support this as is?
- OMI_audio_emitter
  - Waiting on Norbert to sign CLA before we merge #66
  - Stage 2 vote still waiting on repos to be updated. @robertlong to test and merge these before next week
- Collider extension
  - We should probably break up this work a bit so that we can slowly release parts as physics is a huge area
  - OMI_collider useful on it's own
  - How do we fully define OMI_collider so that it's useful without OMI_physics_body and acts the same across engines?
- Avatar interop group
  - Figure out how to work together on artist workflows and extensions
  - Meetups / workshops in the avatar interop group
  - glTF extension group can learn a lot from their feedback

**Comment by madjin** [comment]
Notes from Lyuma reposted here on request of ifire:

talked about dynamic text rendering. I think it should be just "text", not specify 3D or SDF or 2D. Let implementation choose what type of textbox to use. If you really want 3D polygonal text, just model it in the creator program.

talked about ux3D/OMI who made an implementation in C++ (it's a native viewer for gltf)

brought up web url security. I explained a good amount of what us three you and tim talked about.

physics body extension is very complicated an extensive. can we split up work by just releasing colliders?

finally, retargeting system was discussed. jin said a lot of people have been working on retargeting. I mentioned the godot discussion with tokage
